src/main/java/org/spongepowered/api/scoreboard/objective/displaymode/ObjectiveDisplayModes.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered.org <http://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.scoreboard.objective.displaymode;
+
+/**
+ * {@link ObjectiveDisplayMode}s which cause scores for an
+ * {@link org.spongepowered.api.scoreboard.objective.Objective} to be be
+ * displayed differently.
+ */
+public class ObjectiveDisplayModes {
```

Comment:
>this might be stupid and anti-minecraft's-code-style but what about
>renaming Criteria to Criterion and Criterias to Criteria?

I would support this for the API. We should strive to be as consistent and
clear with our names as possible, and bad plurals just end up confusing
people.

On Mon Jan 26 2015 at 11:07:19 AM gratimax <notifications@github.com> wrote:

> In
> src/main/java/org/spongepowered/api/scoreboard/objective/displaymode/ObjectiveDisplayModes.java
> <https://github.com/SpongePowered/SpongeAPI/pull/299#discussion_r23540736>
> :
>
> > + * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
> > + * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
> > + * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
> > + * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
> > + * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
> > + * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
> > + * THE SOFTWARE.
> > + */
> > +package org.spongepowered.api.scoreboard.objective.displaymode;
> > +
> > +/**
> > + * {@link ObjectiveDisplayMode}s which cause scores for an
> > + * {@link org.spongepowered.api.scoreboard.objective.Objective} to be be
> > + * displayed differently.
> > + */
> > +public class ObjectiveDisplayModes {
>
> public final class ObjectiveDisplayModes { private ObjectiveDisplayModes()
> {} }
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SpongePowered/SpongeAPI/pull/299/files#r23540736>.
>